primitives/file-key-verifier/src/consts.rs

```diff
@@ -0,0 +1,13 @@
+use sp_core::Hasher;
```

Comment:
Not sure about putting these constants here. This is the `file-key-verifier` crate, something abstracted over these constants. This seem more suited to be in the runtime, that makes the `FileKeyVerifier` struct concrete by setting these constants, to what it makes sense in the context of the StorageHub runtime.

Reply:
This was a discussion I had with @links234 to determine how we can unify our common types which the client and runtime depend on to not differ in implementations.